### ISSUE TITLE
Distinguish between commands and outputs in console blocks

### ISSUE THREAD

**Issue by alexronquillo** [newrelic/developer-website#852]
### Is your feature request related to a problem? Please describe

`console` code blocks aren't styled to distinguish between commands and output text. Here's an example of a `console` block rendered on the developer site:

<img width="796" alt="Screen Shot 2020-10-16 at 12 13 34 PM" src="https://user-images.githubusercontent.com/8672090/96283525-53d11380-0faa-11eb-88d8-6051c912d1bd.png">

### Describe the solution you'd like

When I create a `console` block, I would like to be able to distinguish between the command and the output text. Here's an example of how GitHub renders `console` blocks:

```console
$ git help
usage: git [--version] [--help] [-C <path>] [-c <name>=<value>]
           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]
           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]
           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
           <command> [<args>]
```

On top of the styling, it would be really nice to have the **Copy** button only copy the commands (without the prompt).

### Describe why this important to you

Right now, we usually separate the commands from the outputs into two code blocks, whereas it would often be a better experience to combine them, since the output directly relates to the command.

**Comment by jerelmiller** [comment]
Hey @alexronquillo you are in luck! During our innovation week this week, I've added the ability to do exactly this! I'll tag you once that is merged so you can update to use it!

**Comment by alexronquillo** [comment]
Awesome, @jerelmiller!!! 👏 Thank you!

**Comment by jerelmiller** [comment]
@alexronquillo Here is a link to the demo that shows exactly this! https://pr-855.dlyi50rq9kt6c.amplifyapp.com/build-apps/map-pageviews-by-region

See if that addresses your issue. I still need to write some documentation on how to specify output, but this should address this issue. I'll link this to the PR once that is ready to go.

**Comment by alexronquillo** [comment]
@jerelmiller Thanks for showing this. I love the approach and think this really solves this particular problem. I have a few questions about the visuals, but functionally-wise, this is so great!

Questions:

- Are we concerned with consistency in code blocks? These being dark-mode-only and animated in, vs the others which change with the light and dark theme and don't animate at all.
- Are we concerned with showing the mac-style UI buttons? I don't mind them at all, but typically, we crop UI out of screenshots and such, so I wonder if they're necessary here.

**Comment by jerelmiller** [comment]
> Are we concerned with consistency in code blocks?

The terminal component is meant to be visually different than the regular code blocks to try and convey to the user that these are something you should enter in your terminal (hence the "app" look). 

While the example from the link above animates the terminal commands, this is a flag that is passed to the code block by the content author. The default behavior is to leave them static and not animate them.

Terminals are usually dark, so I intentionally kept them dark even on light mode for this reason.

> Are we concerned with showing the mac-style UI buttons?

I did a ton of looking around to see how others represented a "terminal" in the UI, and the mac style buttons seemed to best convey a "this is an application" style interface. Here was what I ended up mimicking: https://codehike.org/mini-terminal

I like them only because it makes it clearer that its meant to represent an app (your terminal), not just some code that you'll be entering in your editor.

Other examples I found:

* https://dribbble.com/shots/9178875-helloworld
* https://dribbble.com/shots/4877607-Animation-Terminal-Session-Liferay-DXP
* https://dribbble.com/shots/6636073-Hypest-for-Hyper
* https://dribbble.com/shots/2345285-404-page-008
* https://dribbble.com/shots/3565234-Terminal-Window-UI

**Comment by alexronquillo** [comment]
Thanks! These reasons make sense, and I'm on board with them.

I'm not sure if we have any examples of this, yet, but one more question about the mac-specific UI... what if we need to show the difference between a mac command and a windows/linux command? For example, installing a dependency using homebrew vs whatever Window's equivalent package manager a user is using?

**Comment by jerelmiller** [comment]
> what if we need to show the difference between a mac command and a windows/linux command?

Thats a really interesting thought! I think this is a great feature request! Would you be willing to open a separate issue for that request? It would be great to use that issue to track and gauge interest in OS-specific UI for the terminal.

**Comment by alexronquillo** [comment]
@jerelmiller done! https://github.com/newrelic/developer-website/issues/863
